src/main/scala/tech/cryptonomic/conseil/util/RouteHandling.scala

```diff
@@ -62,6 +63,15 @@ trait RouteHandling {
     }
     complete(response)
   }
+
+  /** validates platform */
+  protected def validatePlatform(config: PlatformsConfiguration, platform: String): Directive0 = {
+    getNetworks(config, platform) match {
```

Comment:
Idea: would an `if/else` looking for `isEmpty` be easier to follow?

Reply:
I've changed to `if/else`